tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 downloadcache = {toxworkdir}/cache/
-envlist = py2.7-django1.5,py2.6-django1.5,py2.7-django1.4,py2.6-django1.4,py2.7-django1.3,py2.6-django1.3
+envlist = py3.4-django1.9, py3.4-django1.8, py2.7-django1.9,py2.7-django1.8,py2.7-django1.7,py2.7-django1.6,py2.7-django1.5,py2.6-django1.5,py2.7-django1.4,py2.6-django1.4,py2.7-django1.3,py2.6-django1.3
```

Comment:
Latest versions of tox allow to provide list of environments in [more generative way](https://testrun.org/tox/latest/config.html#generating-environments-conditional-settings) using syntax like `py26-django{15,16}, py27-django{15,16,17,18,19}`.

With addition of conditional declarations of requirements we could make this config more concise and less repetitive. If we already change this file I would really appreciate if you could improve it using that approach.

For the reference see example of `tox.ini` files for some other projects:
* [django-userena/tox.ini](https://github.com/bread-and-pepper/django-userena/blob/master/tox.ini)
* [python-gmaps/tox.ini](https://github.com/swistakm/python-gmaps/blob/master/tox.ini)

I think that userena's config could be copied with just minor changes (remove pep8 and coverage envs). It has an advantage of testing every combination of supported Python/Django versions pairs since Django1.4.